modules/ob-release-notes-1-5.adoc

```diff
@@ -0,0 +1,60 @@
+// This module is included in the following assemblies:
+// * about/ob-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+*[id="ob-release-notes-1-5_{context}"]
+= Release notes for {builds-shortname} 1.5
+
+{builds-shortname} 1.5 is now available on {ocp-product-title} 4.16, 4.17, 4.18, and 4.19.
+
+[id="new-features-1-5_{context}"]
+== New features
+
+The following sections highlight what is new in {builds-shortname} 1.5.
+
+* With this update, {builds-operator} deploys Shipwright v0.16 components.
+* With this update, you can use the optional `.spec.nodeSelector`value with `shp` CLI to specify which nodes {builds-shortname} should run on.
```

Comment:
Hi @deerskindoll, 
This `node selector` is used to run the user's build pods, not the builds operator pod.

{builds-shortname} should be replaced with "build run pods"